db/seeds.rb

```diff
@@ -286,11 +286,13 @@ def create_if_none(klass, count = nil)
   }
 
   broadcast_messages.each do |type, message|
+    broadcastable = WelcomeNotification.create
```

Comment:
any reason not to use `create!` here? I suspect if you _did_ use it here, then this would fail, since the `WelcomeNotification` would want/need the `Broadcast` to exist before the `WelcomeNotification` was created

Reply:
I've addressed all of your change requests and I am now continuing to dig into borked Announcements and fix up the spec failures associated with all of the changes that I've made! 👍